src/og/ellipsoid/wgs84.js

```diff
@@ -10,4 +10,4 @@ import { Ellipsoid } from "./Ellipsoid.js";
  * WGS84 ellipsoid object.
  * @type {Ellipsoid}
  */
-export const wgs84 = new Ellipsoid(6378137.0, 6356752.3142);
+export const wgs84 = new Ellipsoid(6378137.0, 6356752.3142); // Equitorial Radius (m), Polar Radius (m)

```

Comment:
This comments is ok:)

Reply:
Thanks. I just what is an ellipsoid 2 days ago! :) Thought it my be useful for some else